django_app/redbox_app/settings.py

```diff
@@ -104,7 +115,6 @@
 
 AUTHENTICATION_BACKENDS = [
     "django.contrib.auth.backends.ModelBackend",
-    "magic_link.backends.MagicLinkBackend",
```

Comment:
this doesnt exist (maybe from another implementation https://pypi.org/project/django-magiclink/ that we aren using?)